bazel/foreign_cc/BUILD

```diff
@@ -0,0 +1,82 @@
+licenses(["notice"])  # Apache 2
+
+load("//bazel:envoy_build_system.bzl", "envoy_package")
+load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
+load("@foreign_cc_platform_utils//:tools.bzl", "NINJA_COMMAND", "NINJA_DEP")
+
+envoy_package()
+
+cmake_external(
+    name = "ares",
+    cache_entries = {
+        "CARES_SHARED": "no",
+        "CARES_STATIC": "on",
+    },
+    cmake_options = ["-GNinja"],
+    lib_source = "@com_github_c_ares_c_ares//:all",
+    make_commands = [
+        NINJA_COMMAND,
+        NINJA_COMMAND + " install",
+    ],
+    static_libraries = ["libcares.a"],
```

Comment:
The output static library files are going to be different on Windows and Linux

Reply:
Ditto.